src/cozy/react/components/InstallationPage/index.jsx

```diff
@@ -101,11 +107,16 @@ const InstallationPage = function({onSkipExtension, initialStep}) {
   }, [hasHint, isVaultConfigured, canAuthWithOIDC])
 
   const canNavigateStepper = !canAuthWithOIDC || isVaultConfigured
+
+  const goBack = () => {
+    setActiveStep(activeStep => activeStep - 1)
```

Comment:
Aren't there some scenario that skip steps? I'm wondering if `activeStep - 1` would work on those scenario.

Reply:
After discussion, there is only one case (extension_installed: true) where we go directly to step configureExtension. But it does not break it.